templates/contribute/index.md

```diff
@@ -17,22 +17,22 @@ If you need to link to Lean 3 specific resources please link there.
 </p>
 </div>
 
-# How to contribute to mathlib
+# How to contribute to mathlib4
 
 Here are some tips and tricks
 to make the process of contributing as smooth as possible.
 
 * Use [Zulip](https://leanprover.zulipchat.com/) to
    discuss your contribution before and while you are working on it.
 * Adhere to the guidelines:
-   - The [style guide](style.html) for contributors.
-   - The explanation of [naming conventions](naming.html).
-   - The [documentation guidelines](doc.html).
+   - The [style guide](https://github.com/leanprover-community/leanprover-community.github.io/blob/lean4/templates/contribute/style.md) for contributors.
+   - The explanation of [naming conventions](https://github.com/adomani/leanprover-community.github.io/blob/patch-2/templates/contribute/naming.md).
+   - The [documentation guidelines](https://github.com/leanprover-community/leanprover-community.github.io/blob/lean4/templates/contribute/doc.md).
```

Comment:
Why did you replace those links to webpages by links to their source code? Please revert that change.